src/core/dev/ring_simple.cpp

```diff
@@ -90,6 +90,12 @@ ring_simple::ring_simple(int if_index, ring *parent, ring_type_t type, bool use_
     , m_gro_mgr(safe_mce_sys().gro_streams_max, MAX_GRO_BUFS)
 {
     net_device_val *p_ndev = g_p_net_device_table_mgr->get_net_device_val(m_parent->get_if_index());
+    BULLSEYE_EXCLUDE_BLOCK_START
+    if (!p_ndev) {
+        throw_xlio_exception("Cannot find netdev for a ring");
```

Comment:
Is this situation possible? Should we have a reference of a netdev if a ring is related to it?

Reply:
This situation doesn't look possible, but coverity generates a warning. So this is just a suppression.

Reply:
Lets add a comment that its done for Coverity, so we don't remove it or think that we obviously have this situation.

Reply:
Done.